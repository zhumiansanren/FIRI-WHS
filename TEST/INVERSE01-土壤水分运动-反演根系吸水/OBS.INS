jif @
@        IOBS        CELL@
l1 [O1]91:96
l1 [O2]91:96
l1 [O3]91:96
l1 [O4]91:96
l1 [O5]91:96
l1 [O6]91:96
l1 [O7]91:96
l1 [O8]91:96
l1 [O9]91:96
l1 [O10]91:96
l1 [O11]91:96
l1 [O12]91:96
l1 [O13]91:96
l1 [O14]91:96
l1 [O15]91:96
l1 [O16]91:96
l1 [O17]91:96
l1 [O18]91:96
l1 [O19]91:96
l1 [O20]91:96
l1 [O21]91:96
l1 [O22]91:96
l1 [O23]91:96
l1 [O24]91:96
l1 [O25]91:96
l1 [O26]91:96
l1 [O27]91:96
l1 [O28]91:96
l1 [O29]91:96
l1 [O30]91:96
