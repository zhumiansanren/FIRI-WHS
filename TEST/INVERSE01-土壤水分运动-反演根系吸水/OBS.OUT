================================================================================
                              农田水热盐运移数值模型                 
                                    FIRI-WHS                                    
                            中国农业科学院农田灌溉研究所          
================================================================================

项目名称: This is a demo of inversing root water uptake.
应力期数量:            1

        IOBS        CELL       DEPTH      PERIOD      PERLEN        TIME        HEAD       MOIST
           1          10        9.50           1      1.0000      1.0000  -8.713E+01      0.2535
           2          20       19.50           1      1.0000      1.0000  -2.046E+02      0.1913
           3          30       29.50           1      1.0000      1.0000  -2.090E+02      0.1900
           4          45       44.50           1      1.0000      1.0000  -2.090E+02      0.1900
           5          60       59.50           1      1.0000      1.0000  -2.090E+02      0.1900
